Spigot-Server-Patches/0698-Add-ExitPortalCreateEvent.patch

```diff
@@ -0,0 +1,45 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: ravkr <rav_kr@interia.pl>
+Date: Fri, 26 Mar 2021 02:25:05 +0100
+Subject: [PATCH] Add ExitPortalCreateEvent
+
+
+diff --git a/src/main/java/net/minecraft/world/level/dimension/end/EnderDragonBattle.java b/src/main/java/net/minecraft/world/level/dimension/end/EnderDragonBattle.java
+index 0ab6319aa3e4e1f5679f37be36999ca56ca2484c..e763b3bbb958491d5faec6df5999deb5d5d27605 100644
+--- a/src/main/java/net/minecraft/world/level/dimension/end/EnderDragonBattle.java
++++ b/src/main/java/net/minecraft/world/level/dimension/end/EnderDragonBattle.java
+@@ -58,6 +58,9 @@ import net.minecraft.world.phys.AxisAlignedBB;
+ import org.apache.logging.log4j.LogManager;
+ import org.apache.logging.log4j.Logger;
+ import io.papermc.paper.event.block.DragonEggFormEvent; // Paper - DragonEggFormEvent
++import io.papermc.paper.event.world.ExitPortalCreateEvent; // Paper - ExitPortalCreateEvent
++import org.bukkit.Location; // Paper - ExitPortalCreateEvent
++import net.minecraft.server.MCUtil; // Paper - ExitPortalCreateEvent
```

Comment:
Imports are not necessary.

Reply:
are you sure? without them it won't compile. (unless I put whole package name before class names)

Reply:
@kashike ?

Reply:
You can use the full name of functions instead of imports, for example instead of `new ExitPortalCreateEvent(location, flag)` you should be able to use `new io.papermc.paper.event.world.ExitPortalCreateEvent(location, flag)`

Reply:
Or maybe kashike meant that it's not necessary to add //Paper comments for imports, as maintainers just let their IDEs sort out imports. But who knows for certain what he meant...

Reply:
that's why I'm waiting for his response... almost one month